index.html

```diff
@@ -759,6 +760,14 @@
                 </ol>
               </li>
             </ol>
+            <div class="note">
+              To avoid leaking information that could fingerprint the user, the
+              user agent SHOULD assign the <var>callbackId</var> based on the
+              identity of the <a>media element</a> alone, and not the identity
+              of the browser profile or any <a>remote playback device</a>.  For
+              example, the user agent could base the <var>callbackId</var> on
+              the timestamp the <a>media element</a> was created.
```

Comment:
nit: I think this example implies the callback ids must be unique across all the media elements within a browser session which is not true. They only must be unique for each media element alone. An example is a counter in the media element's scope, that's set to 0 when the media element is created and is incremented each time its value is returned by ```watchAvailability()``` for this particular element as the callback id.